application/config.json.template

```diff
@@ -110,5 +110,7 @@
         "special": [
         ]
     },
+    "memberCountCategoryName": "Info",
+    "updateMemberCountEveryHour", "12",
```

Comment:
i dont think the time needs a config

Reply:
Ahh, me and squid asked it for it as a config. While, I didn't really care about it, squid leaned towards making it a config. I suppose the main reason being, keeping it as flexible as possible.

Reply:
yeah. so please remove it